public/admin/config.yml

```diff
@@ -388,51 +388,45 @@ collections:
           - label: Metadata description
             name: metadataDescription
             widget: text
-      - name: Products
-        label: 'Products'
-        file: src/cms-content/learn/products.json
+      - label: 'Products: Landing page'
+        name: 'ProductsLanding'
+        file: src/cms-content/learn/products-landing.json
         fields:
           - label: Header
             name: header
             widget: string
           - label: Intro
             name: intro
             widget: markdown
-          - label: Sections
-            name: sections
+            required: false
+          - label: Products List
+            name: productsList
             widget: list
             fields:
               - {
-                  label: Section title,
-                  name: sectionTitle,
+                  label: Product name,
+                  name: productName,
                   widget: string,
                   hint: 'Product name (ex. API)',
                 }
               - {
-                  label: Section ID,
-                  name: sectionId,
+                  label: Product ID,
+                  name: productId,
                   widget: string,
-                  hint: 'Section title in all lowercase, divided by dashes if multi-word (ex: response-simulator)',
+                  hint: 'Product name in all lowercase, divided by dashes if multi-word (ex: response-simulator)',
                 }
               - {
-                  label: Section subtitle,
-                  name: sectionSubtitle,
+                  label: Product subtitle,
+                  name: productSubtitle,
                   widget: text,
-                  hint: 'Short description underneath title',
+                  hint: 'Short few-words-long description underneath product name (ie. Allows you to forecast scenarios)',
                 }
               - {
-                  label: Section description,
-                  name: sectionDescription,
+                  label: Product description,
+                  name: productDescription,
                   widget: markdown,
                   hint: 'Longer product description',
                 }
-              - {
-                  label: Notes,
```

Comment:
also @pnavarrc i took this out because there are 'notes' sections on both the landing page and the individual products pages (per the mocks). i think it might make the most sense to include them in the markdown blocks and just make them/style them as blockquotes in products markdown.